### ISSUE TITLE
update copyrights for GCF to 2014

### ISSUE THREAD

**Issue by ahelsing** [GENI-NSF/geni-tools#463]
This is not fun for anyone, but hopefully it's easy and will save us trouble at the next release.  The following files in the gcf repo have Raytheon BBN copyrights containing the year "2013" (or a range of years ending in "2013").  Please update them so that they say "2014" (or a range of years ending in "2014"):
```
./examples/expirationofmyslices.py
./examples/readyToLogin.py
./examples/myscript.py
./examples/remote-execute.py
./windows_install/setup.py
./acceptance_tests/AM_API/am_api_accept.py
./acceptance_tests/AM_API/am_api_accept_delegate.py
./acceptance_tests/AM_API/am_api_accept_shutdown.py
./acceptance_tests/AM_API/am_api_accept_nagios.py
./acceptance_tests/AM_API/am_api_accept_scaling.py
./src/gcf-am-gib.py
./src/clear-passphrases.py
./src/omni_unittest.py
./src/gen-certs.py
./src/gcf-gch.py
./src/gcf-test.py
./src/gcf-gch-gib.py
./src/delegateSliceCred.py
./src/gcf-am.py
./src/gcf-pgch.py
./src/gcf-ch.py
./src/gcf-proxy-test.py
./src/omni-configure.py
./src/omni.py
./src/stitcher.py
./src/gcf-proxy.py
./src/geni/am1.py
./src/geni/gch.py
./src/geni/ca.py
./src/geni/__init__.py
./src/geni/SecureXMLRPCServer.py
./src/geni/pgch.py
./src/geni/SecureThreadedXMLRPCServer.py
./src/geni/ch.py
./src/geni/config.py
./src/geni/am/proxyam.py
./src/geni/am/resource.py
./src/geni/am/fakevm.py
./src/geni/am/am3.py
./src/geni/am/am2.py
./src/geni/am/aggregate.py
./src/geni/am/gibaggregate/graphUtils.py
./src/geni/am/gibaggregate/resources.py
./src/geni/am/gibaggregate/gib_manager.py
./src/geni/am/gibaggregate/am_gib.py
./src/geni/am/gibaggregate/rspec_handler.py
./src/geni/am/gibaggregate/config.py
./src/geni/util/error_util.py
./src/geni/util/cred_util.py
./src/geni/util/secure_xmlrpc_client.py
./src/geni/util/rspec_schema.py
./src/geni/util/cert_util.py
./src/geni/util/ch_interface.py
./src/geni/util/rspec_util.py
./src/geni/util/urn_util.py
./src/omnilib/amhandler.py
./src/omnilib/chhandler.py
./src/omnilib/handler.py
./src/omnilib/stitchhandler.py
./src/omnilib/stitch/GENIObject.py
./src/omnilib/stitch/workflow.py
./src/omnilib/stitch/RSpecParser.py
./src/omnilib/stitch/defs.py
./src/omnilib/stitch/utils.py
./src/omnilib/stitch/__init__.py
./src/omnilib/stitch/ManifestRSpecCombiner.py
./src/omnilib/stitch/objects.py
./src/omnilib/stitch/VLANRange.py
./src/omnilib/stitch/launcher.py
./src/omnilib/stitch/scs.py
./src/omnilib/xmlrpc/client.py
./src/omnilib/frameworks/framework_sfa.py
./src/omnilib/frameworks/framework_gch.py
./src/omnilib/frameworks/framework_base.py
./src/omnilib/frameworks/framework_pg.py
./src/omnilib/frameworks/framework_gib.py
./src/omnilib/frameworks/framework_of.py
./src/omnilib/frameworks/framework_pgch.py
./src/omnilib/frameworks/framework_gcf.py
./src/omnilib/util/omnierror.py
./src/omnilib/util/files.py
./src/omnilib/util/dates.py
./src/omnilib/util/credparsing.py
./src/omnilib/util/__init__.py
./src/omnilib/util/faultPrinting.py
./src/omnilib/util/dossl.py
./src/omnilib/util/handler_utils.py
./src/omnilib/util/paths.py
./src/omnilib/util/json_encoding.py
./src/omnilib/util/namespace.py
```

 _Imported from trac ticket [#463](http://trac.gpolab.bbn.com/gcf/ticket/463),  created by chaos on 01-06-2014 at 10:53, last modified: 02-13-2014 at 10:08_

**Comment by ahelsing** [comment]
Whoops, sorry, i took the list of files from "master", and should have taken it from "develop", which has some new files.  Corrected the description.

_Trac comment by chaos on 01-06-2014 at 10:55_

**Comment by ahelsing** [comment]
Note that those are not the only files that need editing (e.g. things like LICENSE.txt).

_Trac comment by ahelsing on 01-07-2014 at 10:22_

**Comment by ahelsing** [comment]
Looks good to me.

_Trac comment by chaos on 01-07-2014 at 12:53_

**Comment by luisanevers** [comment]
Verified that all files included in gcf-2.5-021214pm have a 2014 Copyright statement for files modified/owned by Raytheon BBN Technologies.

_Trac comment by lnevers (github user: luisanevers) on 02-13-2014 at 10:08_
